pkg/apis/eventing/v1/broker_types.go

```diff
@@ -74,8 +74,9 @@ type BrokerSpec struct {
 	// +optional
 	Config *duckv1.KReference `json:"config,omitempty"`
 
-	// Delivery is the delivery specification for Events within the Broker mesh.
-	// This includes things like retries, DLQ, etc.
+	// Delivery contains the default delivery spec for each trigger
+	// to this Broker. Each trigger delivery spec, if any, overrides this
+	// global delivery spec.
```

Comment:
Can you update the schema description as well?  https://github.com/knative/eventing/blob/ae17ca31832cf185150fba68fbe0124d18a99172/config/core/resources/broker.yaml#L68-L70